docs/settings.md

```diff
@@ -100,6 +100,9 @@ class GqlAuthSettings()
 > retrieve user based on the decoded token.
 >
 > *This filed must be unique in the database*
+> The default value is the username field of the User model. If you
```

Comment:
ahh this is auto generated :cry: you should add this here 
https://github.com/nrbnlulu/strawberry-django-auth/blob/887bd02ed36139d0d73b13529e0ac0f6ca9650e6/gqlauth/settings_type.py#L245